File: C++/2490. Circular Sentence.cpp
class Solution {
public:
    bool isCircularSentence(string s) {
        if(s[0]!=s.back())
        return false;
        stringstream x(s);
        string c;
        vector<string>a;
        while(x>>c)
        a.push_back(c);
        for(int i=0;i<a.size()-1;i++){
            if(a[i+1][0]!=a[i].back())
            return false;
        }
        return true;
    }
};

File: JAVA/2490. Circular Sentence.java
class Solution {
    public boolean isCircularSentence(String s) {
        String w[]=s.split(" ");
        for(int i=0;i<w.length-1;i++){
            if(w[i+1].charAt(0)!=w[i].charAt(w[i].length()-1))
            return false;
        }
        return(s.charAt(0)==s.charAt(s.length()-1));
    }
}